components/telemetry-operator/internal/validation/plugin_validator.go

```diff
@@ -103,6 +124,26 @@ func checkIfPluginIsValid(content string, pipeline *telemetryv1alpha1.LogPipelin
 	return nil
 }
 
+func validateHTTPOutput(httpOutput telemetryv1alpha1.HTTPOutput) error {
+	if httpOutput.Host.Value != "" && !validHostname(httpOutput.Host.Value) {
```

Comment:
he default of URI is `/` . Here user would need to pass the `/` as URI may be we need to cover this in docs. ?